fastlane/Fastfile

```diff
@@ -428,6 +429,70 @@ platform :android do
     )
   end
 
+  desc 'Builds a prototype build and uploads it to S3'
+  lane :build_and_upload_prototype_build
+    UI.user_error!("'BUILDKITE_ARTIFACTS_S3_BUCKET' must be defined as an environment variable.") unless ENV['BUILDKITE_ARTIFACTS_S3_BUCKET']
+
+    gradle(
+        task: "assembleRelease",
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/pocket-casts-android/blob/14fbc28fd1fb46d68ff9627a26aa8717dc4f1c77/fastlane/Fastfile#L437"></span>Layout/FirstArgumentIndentation: Indent the first argument one step more than the start of the previous line.</td>
    </tr>
  </tbody>
</table>